testit/CMakeLists.txt

```diff
@@ -29,8 +30,8 @@ install(PROGRAMS bin/testit_command.py
 if(CATKIN_ENABLE_TESTING)
   find_package(rostest REQUIRED)
 
-  execute_process(COMMAND docker pull ros:latest)
```

Comment:
Please look at the comments at testit/test/01/config.yaml -- a "clean" system needs to be able to run the tests